doc/man7/EVP_PKEY-RSA.pod

```diff
@@ -129,6 +129,14 @@ The RSA "e" value. The value may be any odd number greater than or equal to
 65537. The default value is 65537.
 For legacy reasons a value of 3 is currently accepted but is deprecated.
 
+=item "prime-check-level" (B<OSSL_PKEY_PARAM_RSA_PRIME_CHECK_LEVEL>) <unsigned integer>
+
+Determines the number of Miller Rabin rounds to perform when doing prime checks.
+The default value of 0 will have an error probability of 2^-100, otherwise it
+uses a higher error probability.
+For OpenSSL providers, this parameter is ignored if the number of primes is
+larger than 2.
```

Comment:
I really don't think we need this.

Reply:
@paulidale Do you have an opinion on this.. I have defaulted to 2^-100 as the error prob. And it uses the table with larger values if the value is 1. (There is a bit of a difference in the round count for the 2 tables). If we dont want this then which table do we want.

Reply:
In a non-adversarial situation, 2<sup>-100</sup> is plenty.
The hassle was adversarial.  Can we guarantee that this is not relevant then?

Since it is keygen, I think we can and should not add it.